Pong/GameObjects/GameObject.cs

```diff
@@ -33,5 +34,13 @@ public void Draw(SpriteBatch spriteBatch)
         {
             spriteBatch.Draw(sprite, new Rectangle(position, size), Color.White);
         }
+
+        public bool CollidesWith(GameObject other)
+        {
+            Rectangle ownRect = new Rectangle(position, size);
+            Rectangle otherRect = new Rectangle(other.position, other.size);
+
+            return ownRect.Intersects(otherRect);
```

Comment:
I have no idea why you are able to access the private members of other here. Imo you shouldn't but 👍